.github/workflows/espresso.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           git clone https://github.com/espressomd/espresso.git
           cd espresso
           mkdir build && cd build
-          cp ../../myconfig.hpp .
+          cp ../maintainer/configs/maxset.hpp myconfig.hpp
```

Comment:
We should probably document at some stage how people should install espresso within SwarmRL. We could try a script approach. Maybe we can open a discussion about it.

Reply:
maybe we just extract the install that CI uses, put it in a bash script and place it next to the requirements.
Different PR though